7nodes with docker-compose

Hi, I was trying to test the 7nodes samples using docker and I get the following error when executing 
> docker-compose up -d

panic: MustNew: Failed to connect to Constellation (/qdata/tm/tm.ipc): lstat /qdata/tm/tm.ipc: no such file or directory

What do I need for the sample to work?. Shouldn't it just work when using docker?.

**Note**: I am using docker for Windows, might that be the problem?.

**UPDATE**: I just followed the steps in Ubuntu and everything is working fine, so I guess it is not compatible with Windows?. If so, maybe it should be explained on the README installation steps.

Thank you very much.

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@ generate new ones using Geth's `account` tool and the `--generate-keys` option f
 ## Getting Started
 The 7nodes example can be run in three ways:
 1. By running a preconfigured Vagrant environment which comes complete with Quorum, Constellation, Tessera and the 7nodes example (__works on any machine__).
-1. By running [`docker-compose`](https://docs.docker.com/compose/) against a preconfigured [compose file](docker-compose.yml) which starts 7nodes example (__works on any machine__).
+1. By running [`docker-compose`](https://docs.docker.com/compose/) against a preconfigured [compose file](docker-compose.yml) which starts 7nodes example (__Windows is not supported__).
 1. By downloading and locally running Quorum, Tessera and the examples (__requires an Ubuntu-based/macOS machine; note that Constellation does not support running locally__)
 
 ### Setting up Vagrant

